### PR TITLE
fix colors not working with instance of Color

### PR DESCRIPTION
**Pull request by farfromrefug**
This fixes the colors not working (crashing) with N `Color` instances.
That s something i really need. 
Did it on dev as you seem to be working there

### CODE REVIEW

File: src/InAppBrowser.android.ts [hunk 1]
```diff
@@ -3,13 +3,13 @@ import Bundle = android.os.Bundle;
 import TextUtils = android.text.TextUtils;
 import Intent = android.content.Intent;
 import Context = android.content.Context;
-import Color = android.graphics.Color;
+// import Color = android.graphics.Color;
```
Comment (jdnichollsc) [inline src/InAppBrowser.android.ts:6]: Please don't comment code, remove this instead

Comment (farfromrefug) [reply src/InAppBrowser.android.ts]: Done

**Comment by jdnichollsc** [comment]
Hello mate, I was wondering if you can put that logic from a new file called `src/utils.common.ts` with a method `parseColor` in order to reuse that instance validation?
Thanks for your help, it's really appreciated! <3

**Comment by farfromrefug** [comment]
You mean the test of instance of ?

**Comment by farfromrefug** [comment]
@jdnichollsc done

**Comment by farfromrefug** [comment]
@jdnichollsc could you make a release with this? really need it

**Comment by jdnichollsc** [comment]
Let me try to do that today, I'm on vacations 😅

File: src/InAppBrowser.android.ts [hunk 2]
```diff
@@ -101,23 +102,30 @@ function setup() {
...
-        const colorString = inAppBrowserOptions[InAppBrowserModule.KEY_TOOLBAR_COLOR];
-        try {
-          builder.setToolbarColor(Color.parseColor(colorString));
-          this.isLightTheme = toolbarIsLight(colorString);
-        } catch (error) {
-          throw new Error(
-                  "Invalid toolbar color '" + colorString + "': " + error.message);
+      let colorString = inAppBrowserOptions[InAppBrowserModule.KEY_TOOLBAR_COLOR];
+      if (colorString) {
+        const color = parseColor(colorString);
```
Comment (jdnichollsc) [inline src/InAppBrowser.android.ts:107]: can you parse the color into the try catch please?

File: src/InAppBrowser.android.ts [hunk 3]
```diff
@@ -101,23 +102,30 @@ function setup() {
...
-      if (inAppBrowserOptions[InAppBrowserModule.KEY_SECONDARY_TOOLBAR_COLOR]) {
-        const colorString = inAppBrowserOptions[InAppBrowserModule.KEY_SECONDARY_TOOLBAR_COLOR];
-        try {
-          builder.setSecondaryToolbarColor(Color.parseColor(colorString));
-        } catch (error) {
-          throw new Error(
-                  "Invalid secondary toolbar color '" + colorString + "': " + error.message);
+      colorString = inAppBrowserOptions[InAppBrowserModule.KEY_SECONDARY_TOOLBAR_COLOR];
+      if (colorString) {
+        const color = parseColor(colorString);
```
Comment (jdnichollsc) [inline src/InAppBrowser.android.ts:121]: same here, please parse the color from try catch

File: src/utils.common.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { Color } from "@nativescript/core";
+
+export function parseColor(color: string | Color) {
+    if (color && !(color instanceof Color)) {
```
Comment (jdnichollsc) [inline src/utils.common.ts:4]: Please use 2 spaces for indentation

**Comment by jdnichollsc** [comment]
Please check my comments 👍

**Comment by farfromrefug** [comment]
@jdnichollsc updated with your comments. You should update your project to use elsint and prettier. Right now it is pretty hard to maintain lint rules in this project. And tslint has been deprecated for quite some time.
If you want we could move your project to @nativescript-community and update it to follow the plugin seed we have there.
A lot better than the default one you are currently using

**Comment by jdnichollsc** [comment]
Oh as I said before from Slack, this project is part of Proyecto26 and Tidelift subscription, and I want to keep that, but let me check to update the linter configuration, I'm always happy to receive suggestions 😊

**Comment by farfromrefug** [comment]
@jdnichollsc oh yes i forget quickly. esling config woould be good!

**Comment by jdnichollsc** [comment]
Hello mate, I don't understand that CI/CD issue: `Module not found: Error: Can't resolve 'utils.common'` 🤔

**Comment by farfromrefug** [comment]
@jdnichollsc not sure don't use CIs.

**Comment by farfromrefug** [comment]
Actually I do! Let me fix it

**Comment by farfromrefug** [comment]
@jdnichollsc should be good now

**Comment by jdnichollsc** [comment]
Thanks mate, check the new version 3.0.1! <3

**Pull request merged**